internal/c/parts/video/font/build.mk

```diff
@@ -0,0 +1,22 @@
+FREETYPE_SRCS := \
+	freetypeamalgam.c
+
+FONT_SRCS := \
+	font.cpp
+
+FONT_STUB_SRCS := \
+	stub_font.cpp
+
+FREETYPE_OBJS := $(patsubst %.c,$(PATH_INTERNAL_C)/parts/video/font/%.o,$(FREETYPE_SRCS))
+
+FONT_OBJS := $(patsubst %.cpp,$(PATH_INTERNAL_C)/parts/video/font/%.o,$(FONT_SRCS))
+
+FONT_STUB_OBJS := $(patsubst %.cpp,$(PATH_INTERNAL_C)/parts/video/font/%.o,$(FONT_STUB_SRCS))
+
+$(PATH_INTERNAL_C)/parts/video/font/%.o: $(PATH_INTERNAL_C)/parts/video/font/%.c
+	$(CC) -O2 $(CFLAGS) -DDEPENDENCY_CONSOLE_ONLY -Wall $< -c -o $@
```

Comment:
For these lines you can get rid of the `-DDEPENDENCY_CONSOLE_ONLY` because none of these files depend on `libqb.h` (which was causing the problems).